interp/stdarg.ml

```diff
@@ -28,7 +28,7 @@ let wit_string : string uniform_genarg_type =
   make0 "string"
 
 let wit_pre_ident : string uniform_genarg_type =
-  make0 ~dyn:(val_tag (topwit wit_string)) "preident"
+  make0 "preident"
```

Comment:
I'm pretty sure this is a semantic feature that we share the dynamic representation. Changing it for sole printing reasons looks completely counter-productive to me. That said, I don't know if there is code relying on this in the wild.

Reply:
I don't know, `preident` is typically used for hints table. I don't remember examples of `Tactic Notation` using `string` (there is one in the test-suite), but I don't see reason to identify them more than to identify `preident` and `ident`.

Reply:
It's probably not that important, but I claim that there merely exists one people actually using that somewhere. No witness, but pretty sure that some ML code makes this assumption.